optimum/exporters/openvino/model_configs.py

```diff
@@ -3266,3 +3269,211 @@ def get_model_for_behavior(self, model, behavior: Union[str, VLMConfigBehavior])
+        elif input_name == "raw_spectrogram":
+            shape = [self.sequence_length, self.batch_size, self.reduction_factor, self.num_mel_bins]
+        elif input_name == "encoder_hidden_states":
+            shape = [self.batch_size, self.sequence_length, self.hidden_size]
+        elif input_name == "spectrogram":
+            shape = [self.batch_size, self.sequence_length, self.num_mel_bins]
+        else:
+            raise ValueError(f"Unsupported input {input_name} for DummySpeechT5InputGenerator")
+
+        return self.random_float_tensor(
+            shape=shape,
+            min_value=0,
+            max_value=1,
+            framework=framework,
+            dtype=float_dtype,
+        )
+
+
+class SpeechT5ConfigBehavior(str, enum.Enum):
+    ENCODER = "encoder"
+    DECODER = "decoder"
+    POSTNET = "postnet"
+    VOCODER = "vocoder"
+
+
+@register_in_tasks_manager(
+    "speecht5",
+    *["text-to-audio", "text-to-audio-with-past"],
+    library_name="transformers",
+)
+class SpeechT5OpenVINOConfig(SpeechT5OnnxConfig):
+    DUMMY_INPUT_GENERATOR_CLASSES = (
+        DummyTextInputGenerator,
+        DummySeq2SeqPastKeyValuesGenerator,
+        DummySpeechT5OpenVINOInputGenerator,
+    )
+
+    def __init__(
+        self,
+        config: "PretrainedConfig",
+        task: str = "text-to-audio",
+        int_dtype: str = "int64",
+        float_dtype: str = "fp32",
+        use_past: bool = True,
+        use_past_in_inputs: bool = True,
+        behavior: SpeechT5ConfigBehavior = SpeechT5ConfigBehavior.ENCODER,
+        preprocessors: Optional[List[Any]] = None,
+        legacy: bool = False,
+    ):
+        super().__init__(
+            config=config,
+            task=task,
+            int_dtype=int_dtype,
+            float_dtype=float_dtype,
+            use_past=use_past,
+            use_past_in_inputs=use_past_in_inputs,
+            behavior=behavior,
+            preprocessors=preprocessors,
+            is_postnet_and_vocoder=False,
+            legacy=legacy,
+        )
+
+    def patch_model_for_export(
+        self, model: Union["PreTrainedModel", "TFPreTrainedModel"], model_kwargs: Optional[Dict[str, Any]] = None
+    ) -> "ModelPatcher":
+        return OVSpeechT5ModelPatcher(self, model, model_kwargs=model_kwargs)
+
+    def add_past_key_values(self, inputs_or_outputs: Dict[str, Dict[int, str]], direction: str):
+        if direction not in ["inputs", "outputs"]:
+            raise ValueError(f'direction must either be "inputs" or "outputs", but {direction} was given')
+
+        if direction == "inputs":
+            decoder_sequence_name = "past_decoder_sequence_length"
+            name = "past_key_values"
+        else:
+            decoder_sequence_name = "past_decoder_sequence_length + 1"
+            name = "present"
+
+        for i in range(self._normalized_config.decoder_num_layers):
+            inputs_or_outputs[f"{name}.{i}.decoder.key"] = {0: "total_batch_size", 2: decoder_sequence_name}
```

Comment:
what is the difference between total_batch_size used for past_key_values and batch_size used in other inputs?

Reply:
I think no difference. We can use `batch_size`.

Reply:
Fixed.